src/p2p/net_node.inl

```diff
@@ -567,8 +569,16 @@ namespace nodetool
     {
       net::get_network_address_host_and_port(addr, host, port);
     }
-    MINFO("Resolving node address: host=" << host << ", port=" << port);
 
+    if (boost::ends_with(host, ".onion"))
```

Comment:
This section is now `net::get_network_address` but without i2p support. Calling that function would require re-splitting host/port (that function does it internally too) if `addr` is a hostname, but this section should be called infrequently anyway.

This section should be easier if `get_seed_nodes()` returns a `std::vector<network_address>`, I'll look into this.